questionnaire/2020/questions.yaml

```diff
@@ -285,6 +285,15 @@
     - 2 to 5 years 
     - More than 5 years
 
+  - title: Does your IT organization have a formal and/or sponsored initiatives in place to expand and improve the use of automation on the network?
+    type: Single choice 
+    id: leadership-support
+    responses:
+    - Yes, this is critical to the top-level IT strategy (proactive)
```

Comment:
For consistency with the following answers: "Yes, this is critical to our IT leadership's strategy"